src/main/java/cc/hyperium/gui/GuiDances.java

```diff
@@ -63,6 +63,18 @@ public GuiDances() {
         this.cancel.put("Yeet", () -> {
 
         });
+        
+        this.handlers.put("Hype", netty -> {
+             Hyperium.INSTANCE.getHandlers().getHypeHandler().hype(Minecraft.getMinecraft().thePlayer.getUniqueID());
+             NettyClient client = NettyClient.getClient();
+             if (client != null && netty) {
+                 client.write(ServerCrossDataPacket.build(new JsonHolder().put("type", "hype").put("hyping", true)));
+             }
+         });
+         this.cancel.put("Hype", () -> {
```

Comment:
uhhh shouldnt this have a stopper like the floss does below

Reply:
yeah it needs one

Reply:
ok I’ll look into it